config/optional/views.view.civiremote_funding_application_list.yml

```diff
@@ -1137,15 +1231,15 @@ display:
               empty_column: false
               responsive: ''
             application_process_id:
-              sortable: false
+              sortable: true
               default_sort_order: asc
               align: ''
               separator: ''
               empty_column: false
               responsive: ''
             funding_case_id:
               sortable: true
-              default_sort_order: desc
+              default_sort_order: asc
```

Comment:
Is there a reason to use `asc`? I think by default the more recent applications should be at the top.

Reply:
switched back to desc